package.json

```diff
@@ -23,9 +23,12 @@
     "async": "^0.9.0",
     "chalk": "^1.0.0",
     "cli-table": "^0.3.1",
-    "commander": "^2.7.1",
+    "commander": "^2.9.0",
     "github-repositories": "^1.1.2",
     "ini": "^1.3.3",
     "superagent": "^1.1.0"
+  },
+  "bin": {
+    "nuke": "index.js"
```

Comment:
Some packages seem to use this command name already: https://packages.ubuntu.com/search?searchon=contents&keywords=nuke&mode=exactfilename&suite=yakkety&arch=any

How about `jst-nuke`?